src/components/VideoPlayer/styled.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
should be able to delete this comment

Reply:
Done, waiting for more input before pushing changes.